addons/account/models/mail_scheduled_message.py

```diff
@@ -0,0 +1,176 @@
+import json
+
+from odoo import api, fields, models, SUPERUSER_ID, _
+from odoo.fields import Datetime
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class MailScheduledMessage(models.Model):
+    _inherit = 'mail.scheduled.message'
+
+    from_move_send_wizard = fields.Boolean(compute='_compute_from_move_send_wizard')
+
+    # ------------------------------------------------------
+    # Compute Methods
+    # ------------------------------------------------------
+
+    @api.depends('notification_parameters')
+    def _compute_from_move_send_wizard(self):
+        for scheduled_message in self:
+            notification_parameters = json.loads(scheduled_message.notification_parameters or '{}')
+            scheduled_message.from_move_send_wizard = notification_parameters.get('from_move_send_wizard')
+
+    # ------------------------------------------------------
+    # Actions
+    # ------------------------------------------------------
+
+    def _post_message(self, raise_exception=True):
+        # OVERRIDE
+        # To handle scheduled messages from account.move.send.wizard specifically
+        remained_scheduled_messages = self.env['mail.scheduled.message']
```

Comment:
Try to already prepare recordsets, easier when reading to know what you do and what is the split
```suggestion
        account_specific = self.filtered(lambda s: (
           s.model == 'account.move' and
           json.loads(s.notification_parameters or '{}').get('scheduled_send_payload')
        ))
        remaining = self - account_specific
```